### PR TITLE
merge changes

### PR DESCRIPTION
**Pull request by ljessicc**
updated support urls

### CODE REVIEW

**Comment by karianna** [comment]
@ljessicc you'll need to rebase and also sign the Eclipse CLA (http://www.eclipse.org/legal/CLA.php)

**Comment by karianna** [comment]
@ljessicc looks like this PR had merge conflicts in it.  You can probably ask for help in resolving conflicts in the build channel.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -6,38 +6,37 @@
...
-1. _docker-build.sh_ added.  This script is invoked for building (Adopt) OpenJDK 
+1. _docker-build.sh_ added.  This script is invoked for building (Adopt) OpenJDK
 binaries in a Docker container.
 1. _sbin/prepareWorkspace.sh_ added
 1. _sbin/colour-codes.sh_ removed to simplify code
-1. _makejdk.sh_ removed - please use _makejdk-any-platform.sh_ or (rarely) 
-_sbin/build.sh_ instead. 
-1. _sbin/common-functions.sh_ removed and its logic split 
+1. _makejdk.sh_ removed - please use _makejdk-any-platform.sh_ or (rarely)
+_sbin/build.sh_ instead.
```
Comment (gdams) [inline docs/CHANGELOG.md:28]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
 ### Docker Support
 
 1. `-D` (long form `--docker`) has been added for building in a docker container.
-1. `-c` (long form `--clean-docker-build`) has been added to build from a clean 
-docker container.
-1. `-i` (long form `--ignore-container`) has been added to ignore existing docker 
+1. `-c` (long form `--clean-docker-build`) has been added to build from a clean
+docker container.
```
Comment (gdams) [inline docs/CHANGELOG.md:78]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
...
-1. New _build-farm/make-adopt-build-farm.sh_ added for the new AdoptOpenJDK 
-Build Farm jenkins pipeline to build Adopt OpenJDK binaries.  Sets the default 
-environment variables that are currently set in individual jobs.  This allows 
-us to now track and version these variables.
-1. New _build-farm/set-platform-specific-configurations.sh_ added for the new 
-AdoptOpenJDK Build Farm jenkins pipeline to build Adopt OpenJDK binaries.  Sets 
+1. New _build-farm/make-adopt-build-farm.sh_ added for the new AdoptOpenJDK
+Build Farm jenkins pipeline to build Adopt OpenJDK binaries.  Sets the default
+environment variables that are currently set in individual jobs.  This allows
+us to now track and version these variables.
```
Comment (gdams) [inline docs/CHANGELOG.md:95]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
...
+Sets the default environment variables for specific platforms that are currently
 set in individual jobs.  This allows us to now track and version these variables.
-1. New _build-farm/sign-releases.sh added for the new AdoptOpenJDK Build Farm 
+1. New _build-farm/sign-releases.sh added for the new AdoptOpenJDK Build Farm
 jenkins pipeline to code sign Adopt OpenJDK binaries (Mac and Windows for now).
-1. _pipelines/build/common/build_base_file.groovy_ added. This co-ordinates the various 
- pipeline builds.
-1._pipelines/build/common/create\_job\_from\_template.groovy_ added. This dynamically 
+1. _pipelines/build/common/build_base_file.groovy_ added. This co-ordinates the various
+ pipeline builds.
```
Comment (gdams) [inline docs/CHANGELOG.md:107]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
...
-pipeline code for each build.
-1. _pipelines/build/openjdk\<version\>\_\<variant\>\_\<nightly\|release\>\_pipeline.groovy_ 
-files added.  These will eventually replace the existing individual jobs with a 
-Pipeline for each version and variant.
-1. _pipelines/build/openjdk\<version\>\_pipeline.groovy_ 
-files added. These define the configurations for the 
-_pipelines/build/common/create\_job\_from\_template.groovy_ to create jobs for a pipeline 
- run.
+1. _pipelines/build/common/openjdk\_build\_pipeline.groovy_ added. This forms the base
+pipeline code for each build.
```
Comment (gdams) [inline docs/CHANGELOG.md:111]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 6]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
...
-Pipeline for each version and variant.
-1. _pipelines/build/openjdk\<version\>\_pipeline.groovy_ 
-files added. These define the configurations for the 
-_pipelines/build/common/create\_job\_from\_template.groovy_ to create jobs for a pipeline 
- run.
+1. _pipelines/build/common/openjdk\_build\_pipeline.groovy_ added. This forms the base
+pipeline code for each build.  
+1. _pipelines/build/openjdk\<version\>\_\<variant\>\_\<nightly\|release\>\_pipeline.groovy_
+files added.  These will eventually replace the existing individual jobs with a
+Pipeline for each version and variant.
```
Comment (gdams) [inline docs/CHANGELOG.md:114]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 7]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
...
- run.
+1. _pipelines/build/common/openjdk\_build\_pipeline.groovy_ added. This forms the base
+pipeline code for each build.  
+1. _pipelines/build/openjdk\<version\>\_\<variant\>\_\<nightly\|release\>\_pipeline.groovy_
+files added.  These will eventually replace the existing individual jobs with a
+Pipeline for each version and variant.  
+1. _pipelines/build/openjdk\<version\>\_pipeline.groovy_
+files added. These define the configurations for the
+_pipelines/build/common/create\_job\_from\_template.groovy_ to create jobs for a pipeline
+ run.
```
Comment (gdams) [inline docs/CHANGELOG.md:118]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 8]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
...
 ### Documentation and Misc
 
 1. _README.md_ updated to reflect new scripts.
 1. _docs/build.md_ added to describe how the build farm utilises the scripts.
 1. _docs/generateBuildMatrix.sh_ added to build a table of build statuses.
 1. _docs/generateTestMatrix.sh_ added to build a table of test statuses.
-1. _docs/images/AdoptOpenJDK_Build_Script_Relationships.png_ added to show script 
-relationship.
+1. _docs/images/AdoptOpenJDK_Build_Script_Relationships.png_ added to show script
+relationship.
```
Comment (gdams) [inline docs/CHANGELOG.md:127]: Please remove this whitspace

File: docs/CHANGELOG.md [hunk 9]
```diff
@@ -75,58 +74,58 @@ Please see _makejdk-any-platform.1_ man page for full details.
...
 1. _docs/generateTestMatrix.sh_ added to build a table of test statuses.
-1. _docs/images/AdoptOpenJDK_Build_Script_Relationships.png_ added to show script 
-relationship.
+1. _docs/images/AdoptOpenJDK_Build_Script_Relationships.png_ added to show script
+relationship.  
 1. _docs/images/sequence.svg_ added to show pipeline workflow.
-1. _.gitignore_ changed to reflect new `workspace` base directory, please check 
-your local .gitignore for the diff.
+1. _.gitignore_ changed to reflect new `workspace` base directory, please check
+your local .gitignore for the diff.
```
Comment (gdams) [inline docs/CHANGELOG.md:130]: Please remove this whitspace

File: docs/CONTRIBUTING.md [hunk 10]
```diff
@@ -43,15 +43,15 @@ If you have a proposed fix for an issue, or an enhancement you would like to mak
...
+Keep the project documentation or tests updated if needed as part of your change, and make that part of the same logical issue and PR.  Open your PR against the master branch of the project.
 
 To keep track of [the pull requests we are managing](https://github.com/AdoptOpenJDK/openjdk-build/pulls) we ask that you follow these guidelines for structuring the pull request title and comment.
 
 ### Pull request title
 
-Use a descriptive title, and if it relates to an issue in our tracker please reference which one.  If the PR is not intended to be merged you should prefix the title with "[WIP]" which indicates it is still Work In Progress.  For example, you may wish to send the PR in for an early review as you work through it.
+Use a descriptive title, and reference those that relate to an issue in our issue tracker. Prefix the PR title with ["WIP] indicating that it is still Work In Progress and not intended for merging. For example, you may wish to send the PR in for an early review as you work through it.
```
Comment (gdams) [inline docs/CONTRIBUTING.md:54]:
```suggestion
Use a descriptive title, and reference those that relate to an issue in our issue tracker. Prefix the PR title with "[WIP]" indicating that it is still Work In Progress and not intended for merging. For example, you may wish to send the PR in for an early review as you work through it.
```

File: docs/README.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Repository for code and instructions for building OpenJDK
```
Comment (gdams) [inline docs/README.md:1]: Not sure about moving this file to docs. I'd prefer it to stay in the root of the repo

File: docs/README.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
...
 
-[![Slack](https://slackin-jmnmplfpdu.now.sh/badge.svg)](https://slackin-jmnmplfpdu.now.sh/)
+[![Slack](https://upload.wikimedia.org/wikipedia/commons/d/d5/Slack_icon_2019.svg)](https://adoptium.slack.com/archives/C09NLQQAV)
```
Comment (gdams) [inline docs/README.md:3]: This doesn't seem to be rendering correctly

File: docs/README.md [hunk 13]
```diff
@@ -10,16 +10,16 @@ AdoptOpenJDK makes use of these scripts to build binaries on the build farm at
...
 
-To do this you will need to have your machine set up with a suitable
-compiler and various other tools available. We set up our machines using
+Have your machine set up with a suitable compiler and other tools available.
+We set up our machines using
 ansible playbooks from the [openjdk-infrastructure](https://github.com/adoptopenjdk/openjdk-infrastructure) repository.
 You can also look at the [dockerfile generator](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/docker/dockerfile-generator.sh) for a list of required packages for Ubuntu.
+  
+Once all of the prerequisites are installed, clone this openjdk-build
+repository (`git clone https://github.com/AdoptOpenJDK/openjdk-build`) and
```
Comment (gdams) [inline docs/README.md:19]:
```suggestion
repository (`git clone https://github.com/adoptium/temurin-build`) and  
```

File: docs/README.md [hunk 14]
```diff
@@ -34,37 +34,28 @@ e.g.
...
-This repository contains several useful scripts in order to build OpenJDK
-personally or at build farm scale.
+This repository contains useful scripts used to build OpenJDK personally or at build farm scale.
 
 1. The `build-farm` folder contains shell scripts for multi configuration Jenkins
 build jobs used for building Adopt OpenJDK binaries.
 1. The `docker` folder contains tools for generating dockerfiles which can be used as part of building
 OpenJDK inside a Docker container.
-1. The `git-hg` folder has now been moved to it's own seperate repository. See [openjdk-mirror-scripts](https://github.com/AdoptOpenJDK/openjdk-mirror-scripts).
+1. The `git-hg` folder has now been moved to it's own seperate repository. See [openjdk-mirror-scripts](https://github.com/adoptium/mirror-scripts).
```
Comment (gdams) [inline docs/README.md:53]:
```suggestion
1. The `git-hg` folder has now been moved to it's own seperate repository. See [mirror-scripts](https://github.com/adoptium/mirror-scripts).
```

File: docs/README.md [hunk 15]
```diff
@@ -295,178 +285,3 @@ This would clone OpenJDK source from <https://github.com/AdoptOpenJDK/openjdk-jd
 to `$HOME/openjdk-jdk11u/src`, configure the build with sensible defaults according
 to your local platform and then build OpenJDK and place the result in
 `/home/openjdk/target/MyOpenJDK11.tar.gz`.
-
```
Comment (gdams) [inline docs/README.md:298]: Was there a reason to remove this metadata?

File: sbin/build.sh [hunk 16]
```diff
@@ -237,9 +237,15 @@ configureVersionStringParameter() {
   fi
 
   addConfigureArg "--with-vendor-name=" "${BUILD_CONFIG[VENDOR]:-"AdoptOpenJDK"}"
+<<<<<<< HEAD
```
Comment (gdams) [inline sbin/build.sh:240]: looks like your rebase went wrong here

**Review by gdams — CHANGES_REQUESTED** [review]
Thanks for putting this together, a few whitespace issues and some other nit picks

File: sbin/build.sh (hunk 16 above)
Comment (ljessicc) [reply sbin/build.sh:240]: @gdams @karianna I think I will have to try asking for help with the rebase

**Comment by karianna** [comment]
@ljessicc ping the build channel in slack and someone should help you within the next few days :-)

**Comment by karianna** [comment]
This has been superseded - thanks for your submission!

**Pull request closed without merging**